src/vicsetup/libvicsetup/defs.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _VIC_DEFS_H
+#define _VIC_DEFS_H
+
+#define VIC_INLINE static __inline__
```

Comment:
Please add doc comments for these macros.  Many are self-explanatory to me, some are not.